charts/hcloud-fip-controller/templates/NOTES.txt

```diff
@@ -0,0 +1,8 @@
+hcloud-fip-controller is now running in the cluster.
+{{- if .Values.configInline }}
+hcloud-fip-controller is now running in your cluster with the specified inline config.
```

Comment:
This would duplicate the first part of the text. Please move the line above the if-statement into the else case